src/lithium/interestingness/timed_run.py

```diff
@@ -132,46 +184,41 @@ def timed_run(
     except subprocess.TimeoutExpired:
         child.kill()
         stdout, stderr = child.communicate()
-        sta = TIMED_OUT
+        status = ExitStatus.TIMEOUT
     except Exception as exc:  # pylint: disable=broad-except
-        print("Tried to run:")
-        print(f"  {cmd_with_args!r}")
-        print("but got this error:")
-        print(f"  {exc}")
+        LOG.error(exc)
         sys.exit(2)
     finally:
         if isinstance(child_stderr, BinaryIO) and isinstance(child_stdout, BinaryIO):
             child_stdout.close()
             child_stderr.close()
     elapsed_time = time.time() - start_time
 
-    if sta == TIMED_OUT:
-        msg = "TIMED OUT"
+    if status == ExitStatus.TIMEOUT:
+        message = "TIMED OUT"
     elif child.returncode == 0:
-        msg = "NORMAL"
-        sta = NORMAL
-    elif 0 < child.returncode < 0x80000000:
-        msg = "ABNORMAL exit code " + str(child.returncode)
-        sta = ABNORMAL
+        message = "NORMAL"
+        status = ExitStatus.NORMAL
+    elif child.returncode != ERROR_CODE and 0 < child.returncode < 0x80000000:
+        message = f"ABNORMAL exit code {child.returncode}"
+        status = ExitStatus.ABNORMAL
     else:
-        # return_code < 0 (or > 0x80000000 in Windows)
-        # The program was terminated by a signal, which usually indicates a crash.
+        # The program was terminated by a signal or by the sanitizer (ERROR_CODE)
         # Mac/Linux only!
-        # XXX: this doesn't work on Windows
         if child.returncode < 0:
-            signum = -child.returncode
+            signum = abs(child.returncode)
```

Comment:
Why switch this? Either way works, just curious.

Reply:
abs() just seems more explicit to me.